scrooge-generator/src/main/resources/nodegen/OWNERS

```diff
@@ -0,0 +1,2 @@
+nnance
```

Comment:
mind omitting this file? its relevant internal to Twitter, but doesn't apply outside.